examples/k8s/traefik-ds.yaml

```diff
@@ -37,11 +37,10 @@ spec:
           containerPort: 80
           hostPort: 80
         - name: admin
-          containerPort: 8081
+          containerPort: 8080
```

Comment:
Honestly, according to the kubernetes Configurations best practices documentation (https://kubernetes.io/docs/concepts/configuration/overview/) it clearly states:

`Don’t use hostPort unless it is absolutely necessary`. They recommend using `kubectl proxy` or `kubectl port-forward`. I think if we are providing an example for new users to try, we should at least follow best practices. MANY users have been bitten by hostport usage.